src/disk.c

```diff
@@ -875,6 +873,30 @@ static int disk_read(void) {
 #endif
   } /* while (fgets (buffer, sizeof (buffer), fh) != NULL) */
 
+  /* Remove disks that have disappeared from diskstats */
```

Comment:
Are newly discovered disks also added to diskstats?

Reply:
Yes, in L712: https://github.com/collectd/collectd/pull/2551/files#diff-352f7d19b14cf77c9be0554e611634faR712

Reply:
Thanks (I did the review from my phone which makes reading code hard ;)